src/common/readout/vmm3/VMM3Config.cpp

```diff
@@ -116,8 +120,9 @@ void VMM3Config::loadAndApplyCalibration(std::string CalibFile) {
   try {
     calib_root = from_json_file(CalibFile);
   } catch (...) {
-    LOG(INIT, Sev::Error, "Error loading json calibration file");
-    throw std::runtime_error("Error loading json calibration file");
+    std::string Message = "Error loading json calibration file";
```

Comment:
Following the practice above, I would add a parameter to this error message with the name of the file at least.

Reply:
Agree